schema/layers/visual-layer.json

```diff
@@ -19,7 +19,20 @@
                     "title": "Auto Orient",
                     "$ref": "#/$defs/values/int-boolean",
                     "default": 0,
-                    "description": "If 1, The layer will rotate itself to match its animated position path"
+                    "description": "If 1, the layer will rotate itself to match its animated position path"
+                },
+                "hasMask": {
```

Comment:
Consider adding a conditional to ensure hasMask and masksProperties are in sync (either both set or both unset).

Reply:
When we discussed this we said we would not include `hasMasks` in the specs

Reply:
Removed `hasMasks`.